README.md

```diff
@@ -106,6 +111,11 @@ You can then import your settings in your project like this:
 
     from secure_scaffold.config import settings
 
+### Users 
+
+To use the Users class you will need to enable IAP on your App Engine instance. 
+
+This is so App Engine will send the correct headers.
```

Comment:
Probably worth adding a bit more detail here. It's technically IAP which sends the headers - could provide the docs link as well: https://cloud.google.com/iap/docs/signed-headers-howto